tufaceous/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "tufaceous"
 version = "0.1.0"
 edition = "2021"
+license = "MPL-2.0"
 publish = false
```

Comment:
We may also want to ensure `publish = false` is present everywhere too (it prevents an accidental `cargo publish`). Although the usefulness of that for us is debatable, so probably not for this PR. (It's in tufaceous because I have a habit of adding it to every Cargo.toml I `init`.)